neo/Storage/Interface/DBInterface.py

```diff
@@ -2,11 +2,37 @@
 from neo.SmartContract.Iterable import EnumeratorBase
 from neo.logging import log_manager
 
-logger = log_manager.getLogger('db')
+logger = log_manager.getLogger()
 
 
-class DBCollection:
-    DB = None
+class DBProperties:
+
+    """ Store the different DB properties
+    Used to pass the relevant information to
+    no.Storage.Implementation.[BACKEND].[BACKEND]DBClassMethods.openIter
+
+    Args:
+        prefix (str, optional): Prefix to search for.
+        include_value (bool, optional): include vale used for creating an
```

Comment:
```suggestion
        include_value (bool, optional): include value used for creating an
```

Reply:
I don't understand the description of the argument. Unless my IDE is failing me I only see it used in creation of `DBProperties` but I don't see any logic that actually does anything with it later on?

Reply:
I added an improved description, the old one was not really clear. Actually, I added the `DBProperties` class for convenience and to have a single object you can pass to multiple database methods that might need the same or similar information, for now it's just used for `openIter`. `include_value` is currently only set to `False` in `_BuildCollectionKeys` in the `DBInterface`, other than that it's always `True`.